ewah/hooks/shopify.py

```diff
@@ -208,22 +212,52 @@ def get_data(
         headers = {
             "X-Shopify-Access-Token": self.conn.password,
         }
+
+        # for endpoints that need ids
+        ids_list = []
+        if shopify_object == "inventory_levels":
+            # call location ids
+            locations_url = self._BASE_URL.format(
+                shop=shop_id,
+                version=version,
+                object="locations",
+            )
+            req_kwargs = {
+                "headers": headers,
+            }
+            response = requests.get(locations_url, **req_kwargs)
```

Comment:
Suggestion (non-blocking): No need to do the whole `req_kwargs` stuff here :)

```suggestion
            response = requests.get(locations_url, headers=headers)
```